### DISCUSSION TITLE
When will AI answers become available in the self-hosted version (Enterprise)?

### DISCUSSION THREAD

**Discussion by EduNetEurope** [outline/outline#6893] [Self Hosting]
AI answers was released on 3 January, 2024 for the cloud version.
Are there any plans to release it for the self-hosted version as well - I supposed under a Business oder Enterprise license?

**Comment by tommoor** [comment]
It has been available in the licensed editions since **v0.75.0** – though, additional setup is required:
https://docs.getoutline.com/s/hosting/doc/openai-iiTYCN9Nct
